common/src/main/java/com/wynntils/features/user/map/MapFeature.java

```diff
@@ -37,6 +39,21 @@ public class MapFeature extends UserFeature {
     @Config
     public float poiScale = 1f;
 
+    @Config(subcategory = "Remote Players")
+    public boolean renderRemoteFriendPlayers = true;
+
+    @Config(subcategory = "Remote Players")
+    public boolean renderRemotePartyPlayers = true;
+
+    @Config(subcategory = "Remote Players")
+    public boolean renderRemoteGuildPlayers = true;
+
+    @Config(subcategory = "Remote Players")
+    public HealthTexture remotePlayerHealthTexture = HealthTexture.a;
+
+    @Config(subcategory = "Remote Players")
```

Comment:
What is subcategory

Reply:
I plan to implement them in the config gui, config annotation has it for a while now.